src/app/Components/Form/Form.module.css

```
@@ -0,0 +1,36 @@
+.form {
+  display: grid;
+}
+
+.input__text {
+  display: grid;
+  margin: 0.2em 0 0.4em;
+  font-size: 0.6em;
+  text-transform: uppercase;
+}
+
+.button {
+  border-radius: 1em;
+  border: 0;
+  padding: 0.6em;
+  background-color: var(--text-color);
+  color: var(--text-color-contrast);
+  font-family: var(--font-family-global);
+  transition: background 0.4s;
```

If you like to transition `color` too:

```suggestion
  transition: all 0.4s;
```

And I would play around with timing/easing:

```suggestion
  transition: all 0.4s ease-in-out;
```